prov/verbs/src/verbs_ep.c

```diff
@@ -37,15 +37,32 @@
 static struct fi_ops_msg vrb_srq_msg_ops;
 
 
+void vrb_add_credits(struct fid_ep *ep_fid, size_t credits)
+{
+	struct vrb_ep *ep;
+	struct util_cq *cq;
+
+	ep = container_of(ep_fid, struct vrb_ep, util_ep.ep_fid);
+	cq = ep->util_ep.rx_cq;
+
+	cq->cq_fastlock_acquire(&cq->cq_lock);
+	ep->peer_rq_credits += credits;
```

Comment:
The comment says that peer_rq_credits is protected by the send cq lock, but we use the receive cq lock here.  I think the send cq lock is correct.